spot-client/src/spot-tv/calendars/calendarService.js

```diff
@@ -221,8 +222,10 @@ export class CalendarService extends Emitter {
                 const events = this._updateMeetingUrlOnEvents(formattedEvents);
 
                 if (!this._hasFetchedEvents
```

Comment:
I don't understand why `_hasFetchedEvents` flag is needed exactly. Just don't see why comparing the last fetched events with the new ones is not enough?